.github/workflows/locales-coverage.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
```

Comment:
`v4` is the `Latest` major version of [`actions/checkout`](https://github.com/actions/checkout/) at the time of this pull request's creation.